cmd/mo-service/config.go

```diff
@@ -165,6 +166,35 @@ func (c *Config) createFileService(defaultName string) (*fileservice.FileService
 			return nil, err
 		}
 		services = append(services, service)
+
+		// cache stats
+		if fs, ok := service.(fileservice.CachingFileService); ok {
+			go func() {
+				for range time.NewTicker(time.Second * 10).C {
+					counter := fs.CacheCounter()
+
```

Comment:
I think @fengttt  doesn't want a goroutine per CounterCollection.

According to him, there will be several CounterCollections defined across CN, DN, FileService, LogTail, etc.

He wants one background goroutine to periodically collect the Values from all these CouterCollections and log them one by one.

There will be approx 1000 counters in total.

Definitions:
- counter is "S3ListObjects atomic.Int64"
- counterCollection is "type Counter struct"

Reply:
Yes, I've changed to iterate all fs instances in one goroutine.
Just replace the fmt.Printf with metrics exporter in your following PR.